tomviz/operators/OperatorPython.cxx

```diff
@@ -417,14 +416,29 @@ bool OperatorPython::applyTransform(vtkDataObject* data)
 
   createChildDataSource();
 
-  Python::Object pydata = Python::VTK::GetObjectFromPointer(data);
-
   Python::Object result;
   {
     Python python;
 
     Python::Tuple args(1);
-    args.set(0, pydata);
+
+    // This will look something like:
+    // <function transform_scalars at 0x7fd90c866158>
+    QString name = d->TransformMethod.toString();
```

Comment:
Lets move this to the else block, its only used there?

Reply:
Actually, I was using it to check if the function to be called is "transform_scalars" or "transform", and then pass the appropriate arguments.

It is used in the `if()` and `else if()` conditions.

Reply:
Ok, I see. Its probably better to get the value of `__name__` then you can do a direct comparison? In theory the format of the output of `str(...}` could change, although I agree it will probably always include the function name :smile:

Reply:
Done, this was fixed in the latest commit.

Reply:
Looks great thanks!